src/pages/EditMonitor.vue

```diff
@@ -383,9 +436,32 @@ export default {
 
         },
 
+        isInputValid() {
+            if (this.monitor.body) {
+                try {
+                    JSON.parse(this.monitor.body);
+                } catch (err) {
+                    toast.error(this.$t("The request body is not valid json: ") + err.message);
+                    return false;
+                }
+            }
+            if (this.monitor.headers) {
+                if (!/^([^:]+:.*)([\s]+[^:]+:.*)+$/g.test(this.monitor.headers)) {
+                    toast.error(this.$t("Headers do not have a valid format: \"key: value<new line>key: value<new line>...\""));
```

Comment:
Please add all new i18n keys to `en.js`

Reply:
Is there a script for this or is that a manual process?

Reply:
It's a manual process (correct me if I'm wrong).

You have to add the translations here:
https://github.com/louislam/uptime-kuma/blob/master/src/languages/en.js

And then reference it by it's key. Example:
```
{
    "HeadersDoNotHaveAValidFormat": Headers do not have a valid format: \"key: value<new line>key: value<new line>...\",
}
```

Then usage:
`$t("HeadersDoNotHaveAValidFormat")`

Reply:
fixed